team.sh

```diff
@@ -0,0 +1,75 @@
+echo "Hello..."
+if ! command -v rvm > /dev/null;
+then
+	echo -e "rvm not found...! Install rvm? (y/N) \c";
+	read $REPLY
+	if "$REPLY" = "y";
+	then
+		gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB;
+		\curl -sSL https://get.rvm.io | bash -s stable --ruby;
+		if ! command -v rvm > /dev/null;
+		then
+			echo -e "error while installing rvm. Please try again..."
+			exit 1
+		else
+			echo -e "rvm successfully installed."
```

Comment:
Use the installed rvm to switch to the appropriate ruby version, `2.3.8` afair

Reply:
Rvm automatically detects the ruby version to be installed by the gemfile so I thought it is not necessary to specify the version. Should I change it?